jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/sam/SamInitRunner.kt

```diff
@@ -49,4 +49,16 @@ class SamInitRunner(
         // copy from temp dir to output dir
         VfsUtil.copyDirectory(null, VfsUtil.getChildren(tempDir)[0], outputDir, null)
     }
+
+    companion object {
+        fun testExecutable(path: String): String? {
```

Comment:
We should update this to `validate` and it should also confirm that the version meets our minimum version requirements (0.6.0 I believe) you can use `com.intellij.util.text.SemVer` to help with this.

Reply:
sure, handling in next pr